### ISSUE TITLE
core dumped issue

### ISSUE THREAD

**Issue by danxuhk** [wuhuikai/FastFCN#38]
Dear authors, I run the training script and I found the sementation fault as follows.
![image](https://user-images.githubusercontent.com/8382926/63895981-acc99b00-c9e8-11e9-9c5d-87c4782143bb.png)

What do you think brings this issue? Thanks.

**Comment by wuhuikai** [comment]
I cannot figure it out without further details. Maybe try to run it again : )

**Comment by danxuhk** [comment]
the error comes at this line: x = self.pretrained.conv1(x) in the base.py
If I replace self.pretrained.conv1 with a normal self.conv, it passes. What do you think could be the issue?

**Comment by wuhuikai** [comment]
Can you provide the details of your environment and `base.py` before and after modification?

**Comment by danxuhk** [comment]
I did not change any environment configuration and anything. I use Encnet with JPU and ResNet50. Just used the training sh script to change it...

**Comment by danxuhk** [comment]
My pytorch version is 1.2.0 and CUDA 9.0.

**Comment by wuhuikai** [comment]
Please use `PyTorch==1.0.x`

**Comment by danxuhk** [comment]
I changed to Pytorch==1.0.0. The problem is still there. I did not change anything on the base.py...
When I change BatchNorm2d with torch.nn.BatchNorm2d, it passes. The issue is due to the batch norm layer. Any idea how could solve this?

**Comment by wuhuikai** [comment]
Have you reinstalled `encoding` after changing to `1.0.0`? If you did, then I think everything is the same as my environment :(
Ubuntu 16.04, Python 3.5, PyTorch 1.0.0, Cuda 9.0

**Comment by danxuhk** [comment]
I reinstalled encoding after changing to 1.0.0. It is a segmentation fault caused by using encoding.nn.BatchNorm2D.

**Comment by wuhuikai** [comment]
I have no idea why there's a segmentation fault since you have the same configurations as mine :(
But for a reference, there's a [solution](https://github.com/zhanghang1989/PyTorch-Encoding/issues/158#issuecomment-456824091) in the original `Encoding` repo.

**Comment by wuhuikai** [comment]
@danxuhk The code in branch `latest` is now ready, which uses official `SyncBatchNorm` and can run on all OS with `PyTorch>=1.1.0`.
